MANIFEST.in

```diff
@@ -1,4 +1,7 @@
 include README.md
 include LICENSE
+include HISTORY.rst
 include requirements.txt
+
 recursive-include socorrolib *.py
+recursive-include socorrolib *.tst
```

Comment:
What's that? I don't see any .tst file in this repo. Is it general good practice?